src/2_RTB/VerbaliEsterni/VerbaleEsterno_241123/content.typ

```diff
@@ -0,0 +1,70 @@
+#import "meta.typ": inizio_incontro, fine_incontro, luogo_incontro, company
+#import "functions.typ": glossary, team
+#let participants = csv("participants.csv")
+#let participants_company = csv("participants_company.csv")
+
+= Partecipanti
+/ Inizio incontro: #inizio_incontro
+/ Fine incontro: #fine_incontro
+/ Luogo incontro: #luogo_incontro
+
+== Partecipanti di #team
+#table(
+  columns: (3fr, 1fr),
+  [*Nome*], [*Durata presenza*],
+  ..participants.flatten()
+)
+== Partecipanti di #emph[#company]
+#for member in participants_company.flatten() [
+  - #member
+]
+
+
+= Sintesi Elaborazione Incontro
+
+== Argomenti affrontati
+Di seguito si riportano gli argomenti affrontati durante la riunione:
+
+=== Esposizione dei Progressi avvenuti nello Sprint
+La Propontente ha da subito rivolto la sua attenzione verso i progressi compiuti nel corso dello Sprint appena concluso. 
+
+La discussione si è articolata in due fasi distinte: inizialmente si sono discussi gli aspetti relativi al codice, successivamente si è discusso della parte relativa alla documentazione. 
+
+=== Avanzamenti nel Codice
+Per quanto concerne il codice gli obbiettivi che si erano concordati erano l'*Implementazione di uno script di simulazione* e la *Creazione di un Ambiente Containerizzato*. 
+Il team ha presentato in modo pratico e conciso i risultati ottenuti: è stato attivato un file *Docker Compose*, all'interno del quale lo script di simulazione realizzato in Python, ed eseguito singolarmente in un container, inviava dati simulati a un'istanza di Kafka, anch'essa un container. 
```

Comment:
- "obiettivi"
- non mi dispiace il tecnicismo visto che di fatto e' giusto descrivere brevemente cosa si e' fatt nello sprint/mostrato alla Propnente ma toglierei l'esplicitazione sui singoli containers, della serie: "Il team ha presentato in modo chiaro e compatto i progressi ottenuti: e' stato implementato un file Docker Compose che orchestra un ambiente di esecuzione composto da uno script di simulazione scritto in Python che invia dati simulati a un'istanza di Kafka."

Reply:
1. sostituito ma sarebbe lo stesso
2. Sono d'accordo, Sostituito.